sagemaker-clarify/bias-and-explainability/student_performance_clarify.ipynb

```diff
@@ -0,0 +1,858 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #24.    clarify_processor.run_post_training_bias(
</pre><p>CI testing failed here. Error:</p><pre class="ql-syntax" spellcheck="false">UnexpectedStatusException: Error for Processing job Clarify-Posttraining-Bias-2021-10-04-21-12-32-999: Failed. Reason: ClientError: No Label values are present in the predicted Label Column,Positive Predicted Index Series contains all False values
</pre><p>Please fix this error and ensure that the <code>amazon-sagemaker-examples-pr</code>  GitHub check passes.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2916/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>